test/integration/integration.test.ts

```diff
@@ -706,5 +707,33 @@ describe('Integration tests', (): void => {
         expect(incomingPayment.state).toBe(IncomingPaymentState.Processing)
       })
     })
+    describe('Card-POS Payments', () => {
+      let testActions: TestActions
+
+      beforeAll(async () => {
+        testActions = createTestActions({ sendingASE: c9, receivingASE: hlb })
+      })
+
+      test('POS payment', async () => {
```

Comment:
Maybe we can call this something like "Completes payment successfully", since itll be a happy-path case beside non-happy path cases